src/components/Attachments/AttachmentCarousel/index.js

```diff
@@ -224,11 +223,29 @@ AttachmentCarousel.propTypes = propTypes;
 AttachmentCarousel.defaultProps = defaultProps;
 
 export default compose(
+    // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
     withOnyx({
         reportActions: {
             key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`,
             canEvict: false,
         },
+        parentReport: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT}${report ? report.parentReportID : '0'}`,
+        },
+        parentReportActions: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report ? report.parentReportID : '0'}`,
+            canEvict: false,
+        },
+    }),
+
+    // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
+    withOnyx({
```

Comment:
Why do these need to be separate? Can't we combine them into one and save the error?

Reply:
This is because it's how dependent Onyx keys work. However, with the latest Onyx update, this is no longer necessary and I'll update this.